src/components/Hero/__tests__/__snapshots__/hero.test.tsx.snap

```diff
@@ -16,26 +16,22 @@ exports[`Hero component renders correctly 1`] = `
     <div
       className="download-lts-container"
     >
-      <button
-        className="download-lts-cta t-body1"
-        type="button"
+      <a
+        href="https://nodejs.org/dist//node-.tar.gz"
       >
-        Download Node (LTS)
-      </button>
+        <button
```

Comment:
Buttons shouldn't be nested inside links https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

Reply:
ooh yeaa 😕!

I tried it with:

```js
<a className="download-lts-cta t-body1" href={ltsVersionUrl}>
  Download Node (LTS)
</a>
```

But styles changes:

<img width="429" alt="Screen Shot 2020-05-03 at 00 05 09" src="https://user-images.githubusercontent.com/5341334/80898466-c89ce380-8cd1-11ea-9885-4c7922ace204.png">

Also fixing `Learn Node` button too with:

```js
<Link className="learn-cta t-body1" to="/learn">
  Learn Node
</Link>
```

And styles are not the same too  😕.

<img width="389" alt="Screen Shot 2020-05-03 at 00 06 39" src="https://user-images.githubusercontent.com/5341334/80898495-16b1e700-8cd2-11ea-8eb3-171819edd3fe.png">

Looking into CSS to could send a PR with the fix for it.

Reply:
https://github.com/nodejs/nodejs.dev/pull/655/files#diff-a3053da7eac1859c5b6641368591672dR12-R24 for some inspiration

Reply:
Thanks @benhalverson! Just a quick question, in this link https://www.figma.com/proto/lOxAGGg5KXb6nwie7zXkz6/NJ---Design-System?scaling=min-zoom&node-id=1338%3A13127, is possible to could see design details as font sizes, color, size, etc?

Reply:
Yes click on the figma icon in the top left and signup / signin 
You can then see the artboard with those details. The colors fonts and size are already in tokens.scss

Reply:
@nschonni @benhalverson I have sent a fix in #675